### DISCUSSION TITLE
Documentaion specyfic to Webpack and Vite is broken

### DISCUSSION THREAD

**Discussion by Random-nobody** [quasarframework/quasar#13945] [General - Components / Directives / etc] [answered]
Most of documentation's search results are leading to 404 page. It looks like every page that is specific to Webpack or Vite is broken. Even thought search results are showing parts of the pages so they should exist.

Example:
https://quasar.dev/start/vite-plugin
https://quasar.dev/quasar-cli-vite/lazy-loading#lazy-load-router-pages=

![image](https://user-images.githubusercontent.com/16503469/178713119-109aacad-5cb7-4b66-b07d-0e16b5807240.png)

**Comment by MilosPaunovic** [comment, marked as answer]
Try and clear your browser cache and test again, works just fine in my Chrome.

**Reply by Random-nobody** [reply]
You are right. Somehow my browser cached previous version of Quasar (the picker in the right hand corner) and even after refreshing the page it didn't change. Quite a weird behavior.
